content/docs/user-guide/concepts/dvc-metafiles.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 'DVC Metafiles'
+match: ['DVC files', 'files', 'directories']
+tooltip: >-
+  'DVC [metafiles](/doc/user-guide/concepts/dvc-metafiles) tooltip...'
+---
```

Comment:
p.s. You can list them as comments in each file change if you'd like @jeremydesroches, seems like the easiest approach to me.

p.p.s. This one (dvc-metafiles) doesn't have SEO metadata yet.

Reply:
Ping @jeremydesroches

Reply:
Based on your [comment](https://github.com/iterative/dvc.org/pull/1944#discussion_r536392345) way up there ☝️ it seems like you are saying you don't want to move the Files content or create this item at all. Is that right @jorgeorpinel? If I'm mistaken on this, not a problem to research and create the metadata.

Otherwise, I think I've addressed the remaining feedback so we can start splitting up this PR to work on the pieces in order as you suggested.

Reply:
You're right but now we just need to revert moving content from https://dvc.org/doc/user-guide/dvc-files-and-directories to here. I guess I can just leave it out when I split this PR, so 👍